offline/dist/serviceWorker.js

```diff
@@ -0,0 +1,139 @@
+/* eslint no-restricted-globals: 0 */
```

Comment:
This is the automatically generated file. See offline/serviceWorker.js below for new changes.